.env.sample

```diff
@@ -13,7 +13,7 @@ POKTROLLD_IMAGE_TAG_OFF_CHAIN=0.0.10 # latest
 # using cosmovisor. That value can be adjusted **after** the upgrade, though not necessary for normal operations.
 # Should match the value here when synching the full-node from genesis: 
 # https://github.com/pokt-network/pocket-network-genesis/blob/master/shannon/alpha/testnet-validated.init-version
-POKTROLLD_IMAGE_TAG_VALIDATOR=0.0.9
+POKTROLLD_IMAGE_TAG_VALIDATOR=0.0.10
```

Comment:
This should actually break syncing from scratch on alpha testnet @Olshansk. I'll think how to best address now that we have multiple networks.